webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -63,6 +70,16 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   );
   const [total, setTotal] = useState<Loadable<number>>(NotLoaded);
   const [projectColumns, setProjectColumns] = useState<Loadable<ProjectColumn[]>>(NotLoaded);
+  const [isOpenFilter, setIsOpenFilter] = useState<boolean>(false);
+  const filtersString: string = JSON.stringify(useObservable(formStore.jsonWithoutId));
+  const rootFilterChildren = useObservable(formStore.formset).filterGroup.children;
+
+  const onIsOpenFilterChange = useCallback((newOpen: boolean) => {
+    setIsOpenFilter(newOpen);
+    if (!newOpen) {
+      setTimeout(() => formStore.sweep(), 500);
+    }
```

Comment:
Why do we have a `setTimeout` here?

Reply:
to avoid showing sudden sweeps 

https://github.com/determined-ai/determined/assets/109113616/3434dbca-8172-430f-b271-3c5f6b232a96